.github/CODEOWNERS

```diff
@@ -12,4 +12,4 @@
 # review when someone opens a pull request. Teams should
 # be identified in the format @org/team-name. Teams must have
 # explicit write access to the repository.
-* @climatepolicyradar/tech-devs
+* @climatepolicyradar/software
```

Comment:
@jesse-c the software team exists, but it needs write access to this repo :)

Reply:
Heyo! I've just added software as write on this repo 👍

Reply:
Haha, thank you!